aries-site/src/pages/components/pageheader.mdx

```diff
@@ -0,0 +1,220 @@
+  // represent the example
+  showResponsiveControls={['fullScreen']}
+  screenContainer={{ scale: 0.6 }}
+>
+  <ChildPageHeaderExample />
+</Example>
+
+### Example of Top-Level Page
+
+Top-level pages will not display a reverse anchor because they do not have a parent.
+
+<Example
+  docs="https://v2.grommet.io/pageheader?theme=hpe"
+  // height, width, and screenContainer.scale value chosen to best visually
+  // represent the example
+  showResponsiveControls={['fullScreen']}
+  screenContainer={{ scale: 0.6 }}
+>
+  <TopLevelPageHeaderExample />
+</Example>
+
+## Page-Level Actions
+
+Page-level actions are actions that affect the entire page or kick-off another task flow.
+
+**All actions should:**
+
+- Have clear and concise button labels.
+- Use verb+noun format to provide sufficient context on action expectations (i.e., Edit Profile). See [button labeling]() for more guidance.
+- Be right-aligned with the page content.
+- Follow [button ordering guidance](/components/button#button-ordering) for right-aligned buttons.
+
+**Additional Notes:**
+
+- A primary action should only be used if the primary purpose of the page is to faciliate performing that action. Most pages will not have a primary action.
+- Do not include actions related to table or list data (such as create, update, and delete) in the page-level actions. These should be placed in the toolbar above the data.
+
+<BestPracticeGroup>
+  <Example
+    bestPractice={{
+      type: 'do',
+      message:
+        'Include actions that affect the entire page in the page-level actions. Place actions related to table or list data (such as create, update, and delete) in the toolbar.',
+    }}
+    width="100%"
+    showResponsiveControls={[]}
+    screenContainer={{ scale: 0.6 }}
+  >
+    <PageHeaderActions />
+  </Example>
+  <Example
+    bestPractice={{
+      type: 'dont',
+      message: `Don't include actions related to table or list data (such as create, update, and delete) in the page-level actions.`,
+    }}
+    width="100%"
+    showResponsiveControls={[]}
+    screenContainer={{ scale: 0.6 }}
+  >
+    <PageHeaderActions bestPractice={false} />
+  </Example>
+</BestPracticeGroup>
+
+### Responsive Behavior of Actions
+
+Follow this guidance on how actions should progressively collapse based on their priority.
+
+**A primary action should:**
+
+- Only be used if the primary purpose of the page is the perform that action. Most pages won't have a primary button.
+- Always remain visible, if present. At `xsmall` and `small` breakpoints, the primary button should appear beneath the subtitle and be left-aligned with page content.
+
+**Secondary and default actions:**
+
+- Should remain visible if there is space but collapse into an action menu if there is not enough space.
+- Default buttons should collapse first because they are lower priority than secondary buttons.
+
+Resize your window to see the responsive behavior of the actions.
```

Comment:
I see this now but at first glance I missed this indication that you need to resize the window